### PR TITLE
added packages/linwrap/linwrap.0.0.1

### PR DESCRIPTION
**Pull request by UnixJunkie**

### CODE REVIEW

**Comment by UnixJunkie** [comment]
Some CI tests look stalled: "Waiting for resource in "docker""

File: packages/linwrap/linwrap.0.0.1/opam [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+bug-reports: "https://github.com/UnixJunkie/linwrap/issues"
+dev-repo: "git+https://github.com/UnixJunkie/linwrap.git"
+license: "BSD-3"
+build: ["dune" "build" "-p" name "-j" jobs]
+depends: [
+  "base-unix"
+  "batteries"
+  "cpm"
+  "dolog" {>= "4.0.0"}
+  "dune"
```
Comment (kit-ty-kate) [inline packages/linwrap/linwrap.0.0.1/opam]:
```suggestion
  "dune" {>= "1.10"}
```
According to the `dune-project` file, this package requires at least dune 1.10

**Comment by UnixJunkie** [comment]
I included your suggestion.
I just hope that the dune-project file puts the lowest possible constraint on the dune-version.

**Comment by UnixJunkie** [comment]
The Travis tests look OK.
The docker-related tests look like they are in trouble; but probably not because of my PR.

**Comment by kit-ty-kate** [comment]
Thanks, sorry for the wait

**Pull request merged**